pkg/scheduler/framework/plugins/podtopologyspread/scoring.go

```diff
@@ -256,3 +272,11 @@ func getPreScoreState(cycleState *framework.CycleState) (*preScoreState, error)
 	}
 	return s, nil
 }
+
+// topologyNormalizingWeight calculates the weight for the topology, based on
+// the number of values that exist for a topology.
+// Since <size> is at least 1 (current node), and k8s supports 5k nodes, result
```

Comment:
I think in the corner case that all nodes don't have required topologyKeys present, size is 0. So the lower bound should be `math.Log(0+2) = 0.69`.

Reply:
We don't actually care about that case because we just return 0 score.
But I put that in the comment